inference/server/export.py

```diff
@@ -170,7 +170,7 @@ async def fetch_eligible_chats(session_generator, filters: list[Any]) -> list[Db
     filters.append(DbChat.allow_data_use)
     async with session_generator() as session:
         query = (
-            sqlmodel.select(DbChat)
+            sqlmodel.select(entity_0=DbChat)
```

Comment:
I think some of the named arguments are definitely helpful, but others (like this one) are not useful and imo detract from readability

Maybe we could have a slightly more case-by-case approach with some of the less useful ones taken out

Reply:
removing for all functions call where only one argument is passed ?

Reply:
Yeah, something like that would make sense to me